simple_pdf_generator/src/lib.rs

```diff
@@ -256,27 +242,6 @@ pub async fn generate_pdf(
         SimplePdfGeneratorError::BrowserError(format!("Cannot inject the js: {}", e))
     })?;
 
-    if !template.tables.is_empty() {
```

Comment:
Why you removed the `tables` feature if in the `derive` is still present? I would like to keep this feature in order to have a more coherent behavior across the library

Reply:
That makes sense! I removed it to make it work in my tests, but forgot to readd it. Will be readding it